WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemColorsScreen.kt

```diff
@@ -0,0 +1,134 @@
+        )
+    ) {
+        item {
+            val list: List<ColorOption> = listOf(
+                ColorOption("Primary Foreground", MaterialTheme.colorScheme.primary),
+                ColorOption("Primary Background", MaterialTheme.colorScheme.primaryContainer),
+                ColorOption("Secondary Foreground", MaterialTheme.colorScheme.secondary),
+                ColorOption("Secondary Background", MaterialTheme.colorScheme.secondaryContainer),
+                ColorOption("Tertiary Foreground", MaterialTheme.colorScheme.tertiary),
+                ColorOption("Tertiary Background", MaterialTheme.colorScheme.tertiaryContainer),
+                ColorOption("Quartenary", MaterialTheme.colorScheme.quartenary),
+                ColorOption("Brand", MaterialTheme.colorScheme.brand),
+                ColorOption("Brand Background", MaterialTheme.colorScheme.brandContainer),
+                ColorOption("Error", MaterialTheme.colorScheme.error),
+                ColorOption("Warning", MaterialTheme.colorScheme.warning),
+                ColorOption("WP Foreground", MaterialTheme.colorScheme.wp),
+                ColorOption("WP Background", MaterialTheme.colorScheme.wpContainer),
+                )
+            ColorCardList(list)
+        }
+    }
+}
+@OptIn(ExperimentalStdlibApi::class)
+@Composable
+fun ColorCard (colorName: String, color: Color) {
+    Row (modifier = Modifier.padding(all = 3.dp)) {
+        Box (Modifier.background(MaterialTheme.colorScheme.tertiaryContainer)) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier
+                    .padding(15.dp)
+                    .then(
+                        Modifier
+                            .fillMaxWidth()
+                            .wrapContentSize(Alignment.TopCenter),
+                    )
+            ) {
+                Text(
+                    modifier = Modifier.padding(start = 25.dp, end = 40.dp),
+                    text = colorName,
+                    color = MaterialTheme.colorScheme.primary,
+                )
+                Text(
+                    modifier = Modifier.padding(start = 25.dp, end = 40.dp),
+                    text = "#" + color.value.toHexString().uppercase().substring(0,8),
+                    color = MaterialTheme.colorScheme.secondary
+                )
+            }
+            Column(
+                modifier = Modifier
+                    .padding(15.dp)
+                    .then(
+                        Modifier
+                            .fillMaxWidth()
+                            .wrapContentSize(Alignment.TopStart)
+                    )
+            ) {
+                Box(
+                    modifier = Modifier
+                        .size(45.dp)
+                        .clip(shape = RoundedCornerShape(5.dp, 5.dp, 5.dp, 5.dp))
+                        .background(color)
+                        .border(width = 2.dp,
```

Comment:
Probably, don't need border here

Reply:
I added a border because the background of each row is the `tertiaryBackground` color. Without the border, the row for that element blends in with the background, which wasn't looking so great IMO. I'm fine with keeping it or removing it (or making it thinner) though if we don't like how it looks, open to thoughts